homeworks/david.tereshchenko_DavidTereshchenko/4-basics/basics.js

```diff
@@ -0,0 +1,28 @@
+const firstNumber = +prompt('Enter the first number', '').trim();
+const operator = prompt('Enter mathematics operator (+, -, *, /, %, **,)', '').trim();
+const secondNumber = +prompt('Enter the second number', '').trim();
```

Comment:
It is so many prompts. Use it to get the numbers and render all results on the page